lib/iris/tests/unit/fileformats/grib/load_convert/test_time_range_unit.py

```diff
@@ -22,13 +22,13 @@
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-# import iris tests first so that some things can be initialised
+# import iris.tests first so that some things can be initialised
 # before importing anything else.
 import iris.tests as tests
 
 from cf_units import Unit
-
 from iris.exceptions import TranslationError
+
 from iris.fileformats.grib._load_convert import time_range_unit
```

Comment:
Admittedly I had been ignoring these extra lines, but I actually they should be removed.